test/onnx/test_operators.py

```diff
@@ -518,6 +518,13 @@ def forward(self, x, y):
         BigModule()(*inp)
         self.assertONNX(BigModule(), inp)
 
+    def test_randn(self):
+        x = torch.randn(1, 2, 3, 4)
+
+        class MyModule(Module):
+            def forward(self, x):
+                return torch.randn(1, 2, 3, 4) + x
```

Comment:
looks like we can just use the lambda expression like other tests?

Reply:
sure!